reference/resource-definitions.md

```diff
@@ -66,6 +66,25 @@ end
 {% endhighlight %}
 
 
+## Parent Resource
+
+You can define the parent of a resource by using the `parent` directive. This will configure the resource to use its parent's `canonical_path` as the base for all of its actions (in addition to any `prefix` you may define on this resource).
+
+Additionally, any parameters in the parent's route will also be applied as defaults in the child. The last route parameter is assumed to be an 'id'-type parameter, and is prefixed with the parent's snake-cased singular name. I.e., `id` from a `Blog` parent will be renamed to `blog_id`. Any other parameters are copied unchanged.
+
+This behavior can be overridden by providing a mapping hash of the form `{parent_name => child_name}` to the `parent` directive. 
+
+For example, to define a `Posts` subresource of the above `Blogs` resource:
+
+{% highlight ruby %}
+class Posts
+  include Praxis::ResourceDefinition
+  
+  parent Blogs
+end
+{% endhighlight %}
+
+
```

Comment:
I would add an action in the snippet example above too. I.e.,
```
action :show do
  routing { get '/:id' }
end
```
And then add the following paragraph sentences:

This would result in the `:show` action responding to the following path `/blogs/:blog_id/posts/:id`, due to the canonical path of the `Blogs` resource being `/blogs/:id`.  

To achieve a custom parent parameter we could have used `parent Blogs {:id => :parent_id}` instead, which would have resulted in the following path: `/blogs/:parent_id/posts/:id`